Xamarin.Forms.Platform.WPF/Renderers/RadioButtonRenderer.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System.ComponentModel;
+using System.Windows;
+using System.Windows.Media;
+using WThickness = System.Windows.Thickness;
+
+namespace Xamarin.Forms.Platform.WPF
+{
+	public class RadioButtonRenderer : ViewRenderer<RadioButton, FormsRadioButton>
+	{
+		bool _fontApplied;
+		FormsRadioButton _button;
+
+		public bool IsDisposed { get; private set; }
+
+		protected override void OnElementChanged(ElementChangedEventArgs<RadioButton> e)
+		{
+			base.OnElementChanged(e);
+
+			if (e.NewElement != null)
+			{
+				if (Control == null)
+				{
+					_button = new FormsRadioButton();
+
+					_button.Click += OnButtonClick;
+					_button.AddHandler(System.Windows.Controls.Button.ClickEvent, (RoutedEventHandler)OnPointerPressed, true);
+					_button.Loaded += ButtonOnLoaded;
+					_button.Checked += OnRadioButtonCheckedOrUnchecked;
+					_button.Unchecked += OnRadioButtonCheckedOrUnchecked;
+
+					SetNativeControl(_button);
+				}				
+
+				UpdateContent();
+
+				//TODO: We may want to revisit this strategy later. If a user wants to reset any of these to the default, the UI won't update.
+				if (Element.IsSet(VisualElement.BackgroundColorProperty) && Element.BackgroundColor != (Color)VisualElement.BackgroundColorProperty.DefaultValue)
+					UpdateBackground();
+
+				if (Element.IsSet(RadioButton.TextColorProperty) && Element.TextColor != (Color)RadioButton.TextColorProperty.DefaultValue)
+					UpdateTextColor();
+
+				if (Element.IsSet(RadioButton.BorderColorProperty) && Element.BorderColor != (Color)RadioButton.BorderColorProperty.DefaultValue)
+					UpdateBorderColor();
+
+				if (Element.IsSet(RadioButton.BorderWidthProperty) && Element.BorderWidth != (double)RadioButton.BorderWidthProperty.DefaultValue)
+					UpdateBorderWidth();
+
+				if (Element.IsSet(RadioButton.CornerRadiusProperty) && Element.CornerRadius != (int)RadioButton.CornerRadiusProperty.DefaultValue)
+					UpdateBorderRadius();
+
+				if (Element.IsSet(RadioButton.PaddingProperty) && Element.Padding != (Thickness)RadioButton.PaddingProperty.DefaultValue)
+					UpdatePadding();
+
+				UpdateFont();
+				UpdateCheck();
+			}
+		}
+
+		void ButtonOnLoaded(object o, RoutedEventArgs routedEventArgs)
```

Comment:
Don't access anything related to the native control when loaded. If not necessary, remove the method.

Reply:
Done